dts/arm/nordic/nrf52840.dtsi

```diff
@@ -137,6 +137,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cc310: crypto@5002b000 {
+				compatible = "arm,cryptocell-310";
+				reg = <0x5002B000 0x1000>;
+				interrupts = <42 1>;
```

Comment:
I suppose the "interrupt" is Nordic-specific, so if this is possible, I would put it on the "parent" object

Reply:
I don't think so. ARM Cryptocell-310 has an interrupt source, so it is ARM Cryptocell-310 specific. Actual number of interrupt is chip specific. Generally it is exactly the same situation as in case of `reg`.